data/curated/new_submission/intro.md

```diff
@@ -0,0 +1,20 @@
+The dataset this week explores seismic events detected at the famous Mount Vesuvius in Italy. It comes from the [Italian Istituto Nazionale di Geofisica e Vulcanologia (INGV)](https://www.ingv.it/)'s [Data Portal](https://data.ingv.it/en/) and can be explored
+along with other seismic areas on [the GOSSIP website](https://terremoti.ov.ingv.it/gossip/vesuvio/index.html). The raw data was saved as individual CSV files from the GOSSIP website and some values were translated from Italian to English.   
+
+
+> The Open Data Portal of Istituto Nazionale di Geofisica e Vulcanologia (INGV) gives public access to data resulting from institutional research activities in the fields of Seismology, Volcanology, and Environment.  
+
+Some information about Mount Vesuvius [from INGV](https://www.ingv.it/somma-vesuvio):  
+> Location: Campania, 40°49′18.01″N, 14°25'33.57” E
+> Maximum height: 1281 m above sea level
+> Total surface area: ≈115-150 km2
+> Type of volcano: stratovolcano
+> Start of eruptive activity: <39,000 years
+> Last eruption: 1944 (lasted about 10 days)
+> Activity status: quiescent (not active, but is still registering seismic activity)
```

Comment:
```suggestion
- Location: Campania, 40°49′18.01″N, 14°25'33.57” E
- Maximum height: 1281 m above sea level
- Total surface area: ≈115-150 km2
- Type of volcano: stratovolcano
- Start of eruptive activity: <39,000 years
- Last eruption: 1944 (lasted about 10 days)
- Activity status: quiescent (not active, but is still registering seismic activity)
```
The `>`ed text all merged into one line when I previewed. I think this will keep the idea you were looking for. I'll apply this myself unless I end up with bigger changes for you.